index.html

```diff
@@ -23,13 +25,9 @@
       color: var(--text-color-minus);
     }
 
+    /* XXX temporary hack to make divider not inherit from switch */
     gaia-list .divided {
-      border-left: solid 1px;
```

Comment:
general rules are located in gaia-list, add divider hack to overwrite border-round style inherit from `gaia-switch`